### PR TITLE
PPTP-925 'QuickButton' style position is 'absolute'

### PR DESCRIPTION
**Pull request by PauloPortugal**
Also updated the `README.md` file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,50 @@
 
-# plastic-packaging-tax-returns-frontend
+# Plastic Packaging Tax Returns (PPT)
 
-This is a placeholder README.md for a new repository
+This is the Scala microservice responsible for the transient storage of PPT returns information and PPT Account section,
+which is part of the PPT tax regime, as discussed in this [GovUk Guindance](https://www.gov.uk/government/publications/introduction-of-plastic-packaging-tax/plastic-packaging-tax)
```
Comment (RyanKellyHMRC) [inline README.md:5]: Guindance?

**Review by RyanKellyHMRC — CHANGES_REQUESTED** [review]
One small typo, otherwise good.

File: README.md (hunk 1 above)
Comment (PauloPortugal) [reply README.md:5]: You know from The Government Guindance. Give it a few decades, it will catch up :stuck_out_tongue: eventually

Comment (PauloPortugal) [reply README.md]: This has been addressed.

**Review by RyanKellyHMRC — APPROVED** [review]

**Pull request merged**